CHANGELOG.md

```diff
@@ -2,15 +2,27 @@
 
 ## Unreleased
 
-### Changes
+### Features
 
 - Introduce a `getMe()` method to retrieve the currently logged in user
   from OSIAM 3.0
-- The `BasicUser` class and the method `getCurrentUserBasic()` have been deprecated.
+
+### Changes
+
+- Remove `set` prefixes from single-valued attributes in the resource builders
 
 ### Fixes
 
+- Resources could be changed by using the copy-of constructor of the builders, though they were supposed
+  to be immutable.
 - A `400 BAD REQUEST` response creates a `BadRequestException` instead of the appropriate `ConflictException` 
```

Comment:
I know it's not part of your change, but this line doesn't really make sense, does it?